README.md

```diff
@@ -121,6 +123,17 @@ indexed variable access.
 
 [Back to TOC](#table-of-contents)
 
+lua\_kong\_set\_static\_tag
+-------------------------------------------
+**syntax:** *lua_kong_set_static_tag value;*
+
+**context:** *location(http subsystem)* *server(stream subsystem)*
+
+Add a static tag string for Nginx's `location`(http subsystem) or `server`(stream subsystem) block,
+which can be accessed in Lua land by [`resty.kong.tag.get`](#restykongtagget).
+
```

Comment:
Let's decribe the scope of this tag, it can just reflect the behaviour we observed in tests (derive from server block to location, isolate beteen locations, works in nested location, doesn't work in sub query etc)

Reply:
Add some description.

Another thing, this directive only works in `http/location` or `stream/server`, we can not define tags in `http/server` now. so no deriving from server block.